src/Roslyn.Suppressions/Roslyn.Suppressions/RoslynSettings.cs

```diff
@@ -0,0 +1,86 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using SonarQube.Client.Models;
+
+namespace SonarLint.VisualStudio.Roslyn.Suppressions
+{
+    /// <summary>
+    /// Data class - contains all information that needs to be passed between the main VS
+    /// process and the external Roslyn analysis process
+    /// </summary>
+    public class RoslynSettings
+    {
+        public static readonly RoslynSettings Empty = new RoslynSettings { Suppressions = Enumerable.Empty<SonarQubeIssue>() };
+
+        /// <summary>
+        /// The actual Sonar project key to which the settings relate
+        /// </summary>
+        /// <remarks>Note: this is not necessarily the same as the "settings key" - the settings key is a modified version of the 
+        /// project key without any invalid file characters.</remarks>
+        [JsonProperty("sonarProjectKey")]
+        public string SonarProjectKey { get; set; }
+        
+        [JsonProperty("suppressions")]
+        public IEnumerable<SonarQubeIssue> Suppressions { get; set; }
```

Comment:
For now, the wrapper just returns the existing SonarQubeIssue data i.e. no attempt yet to filter the issues or save less data.